docker/wp-cli/start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+echo "heyyyy"
+
+echo "heyyyy"
+wp core install \
+    --path="/var/www/html" \
+    --url="http://localhost:8001" \
+    --title="Gatsby & WordPress" \
+    --admin_user=admin \
+    --admin_password=secret \
+    --admin_email=admin@admin.com
+
+wp search-replace 'https://gatsbyinttests.wpengine.com' 'http://localhost:8081'
+
+wp user update admin --user_pass="secret"
+# install plugins that the wpgraphql plugins build on
+wp plugin install gutenberg advanced-custom-fields wordpress-seo
```

Comment:
@TylerBarnes we need to pin to specific (provided?) plugin versions here. so we should start with the versions running on the instance where this seed file comes from!

My theory is that this is will resolve the error I'm still seeing wordpress side when querying the schema in the wp-graphql graphiql:

```
PHP Notice:  Undefined index: ID in /var/www/html/wp-content/plugins/advanced-custom-fields/includes/acf-field-functions.php on line 361, referer: http://localhost:8001/wp-admin/admin.php?page=graphiql-ide

```